app/models/user.rb

```diff
@@ -2,6 +2,8 @@ class User < ApplicationRecord
   validates_presence_of :name, :address, :city, :state, :zip
   validates :email, uniqueness: true, presence: true
   validates_presence_of :password_digest, require: true
+  has_many :merchant_employees
```

Comment:
we talked about this earlier, just noting so we don't forget, wondering if we should consider renaming to user, but the user story does explicitly call them merchant_employees so not sure